src/Modules/Windows/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.psd1

```diff
@@ -66,5 +66,6 @@ CmdletsToExport=@("Add-Content",
     "Rename-Computer",
     "Get-ComputerInfo",
     "Get-TimeZone",
-    "Set-TimeZone")
+    "Set-TimeZone",
+    "Get-HotFix")
```

Comment:
Minor nit: For a cleaner history, it would've been be nice if parentheses/commas were placed in such a way that the last line does not require any change for the next addition (i.e. have the closing parenthesis in this case on the next line).